### PR TITLE
visuals for contract dash finished

### PR DESCRIPTION
**Pull request by leeleonard936**
Check it out?

### CODE REVIEW

**Review by Cwagne17 — COMMENTED** [review]
Awesome, lets post a screenshot to see what it looks like.

**Comment by leeleonard936** [comment]
> Awesome, lets post a screenshot to see what it looks like.

image

File: src/app/pages/contract-dashboard/contract-dashboard.component.html [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+  <ng-container matColumnDef="work_id">
+    <th mat-header-cell *matHeaderCellDef> Work Type </th>
+    <td mat-cell *matCellDef="let element">{{element.work_id}}</td>
+  </ng-container>
+
+  <ng-container matColumnDef="butt">
+    <th mat-header-cell  *matHeaderCellDef > Edit </th>
+    <td mat-cell *matCellDef="let row" >
+         <button mat-flat-button class = "editbutt">Edit</button>
```
Comment (leeleonard936) [inline src/app/pages/contract-dashboard/contract-dashboard.component.html]: Add payment is a button, plaiing on adding line click functionality as soon as the contract form is available

**Comment by Cwagne17** [comment]
Can we add a display flex to it and justify the content in between or use grid to space it a certain way

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**